Rovers/Rovers.tex

```diff
@@ -0,0 +1,387 @@
+Because of this the project will require a large amount of prototyping and testing. 
+The team would look at the talent and skills of RIT Space Exploration and answer the question: what caliber of rover are we able to make. 
+It would employ a multidisciplinary team for construction and development. The project would be slated to run for two semesters. 
+The rover would be extremely promotable and would look great at Imagine RIT. 
+\end{abstract}
+
+\label{sec:nomenclature}
+\newcommand{\nomunit}[1]{\renewcommand{\nomentryend}{\hspace*{\fill}#1}}
+\renewcommand{\nompreamble}{}
+
+\nomenclature{RIT}{Rochester Institute of Technology}
+\nomenclature{SPEX}{RIT Space Exploration}
+\nomenclature{PDD}{Project Design Document}
+\nomenclature{URC}{University Robotics Competition}
+\nomenclature{MVP}{Minimum Viable Product}
+
+% Intro
+\section{Introduction}
+\label{sec:introduction}
+
+\IEEEPARstart{R}{obotics}  and by extension rovers are a tremendously important
+part of space exploration. 
+This is also an area that RIT Space Exploration has very little experience with project-wise.
+The purpose of this project is to assess and assert the capability of RIT SPEX regarding the construction and fabrication of a mock-rover and then begin construction. 
+As this is a new area to RIT SPEX, prototyping and experimenting is necessary to most effectively accomplish this goal. 
+This project will look at the unknowns, technical challenges, project management, and member skills of RIT Space Exploration in regard to a University Rover Competition (URC) capable rover and rover construction. 
+The URC is hosted by The Mars Society annually. 
+It is very competitive and requires a very complex and expensive rover that is currently out of scope for SPEX. 
+This rover would need to be capable of full autonomous navigation, soil collection and analysis, and precise robotics. 
+This project is the first step in building the skills for such a team.
+Rovers are important, they are one of the primary ways we explore space and other planets. 
+This project exists to get our members experience with rovers by building such a rover. 
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+The goal of this project is delivering a functioning mock rover.
+We define mock rover as a functioning rover built to specifications for a mock mission as it will not be launched. 
+We define rover as a vehicle for driving over rough terrain, especially one driven by remote control.
+The rover will allow the team to develop skills and knowledge to build a
+more feature completer rover in the future. 
+These skills include rover fabrication and design, autonomous navigation, electrical system design, rover testing, computer vision, and robotics.
+The rover will be testing using events designed like the URC but scaled down to fit the scope of this rover. 
+We will be using time as a metric as well as binary metrics for completion of the parts of the competition. 
```

Comment:
How do you evaluate a function/feature that is present but behaves differently than expected?